src/lib/workspace/Subsetting/SubsetDownloadModal.tsx

```diff
@@ -594,64 +721,63 @@ export default function SubsettingDataGridModal({
             bottom: 25,
             left: 25,
           },
+          size: {
+            height: '100%',
+          },
+        },
+        header: {
+          compact: true,
         },
       }}
     >
-      <H5
-        additionalStyles={{
-          marginTop: 10,
-          marginBottom: 25,
-          fontStyle: 'italic',
-        }}
-        color={gray[700]}
-      >
-        {analysisState.analysis?.displayName}
-      </H5>
-      <div
-        key="Controls"
-        style={{
-          display: 'flex',
-          justifyContent: 'space-between',
-          flexWrap: 'wrap',
-          alignItems: 'center',
-        }}
-      >
-        <div style={{ marginBottom: 15, display: 'flex' }}>
-          <div style={{ marginRight: 25 }} ref={observeEntityDescription}>
-            <span
-              style={{
-                fontSize: 18,
-                fontWeight: 500,
-                color: '#646464',
-              }}
-            >
-              {currentEntity?.displayNamePlural}
-            </span>
-            {currentEntity.filteredCount && currentEntity.totalCount && (
-              <p
+      <div css={{ display: 'flex', flexDirection: 'column', height: '100%' }}>
+        <div css={{ display: 'flex', flexDirection: 'column' }}>
+          <div
+            css={{
+              display: 'flex',
+              justifyContent: 'space-between',
+              marginTop: 10,
+              marginBottom: 15,
+              height: '100%',
+            }}
+          >
+            <div>
+              <span
                 style={{
-                  marginTop: 0,
-                  marginBottom: 0,
-                  color: 'gray',
+                  fontSize: 18,
+                  fontWeight: 500,
+                  color: '#646464',
```

Comment:
gray.600 is close to this color - is there a reason to use specifically #646464 instead?

Reply:
I was just copying what was already there. I can take a look and see if that works.